components/Nav/NavMobile/NavMobile.js

```diff
@@ -34,17 +34,13 @@ export default class NavMobile extends Component {
     return (
       <header className={styles.NavMobile}>
         <Link href="/" prefetch={false}>
-          <button
-            className={classNames(styles.button, styles.logoButton)}
-            type="button"
-            name="dropdown"
-          >
+          <ScreenReaderOnly className={classNames(styles.button, styles.logoButton)}>
```

Comment:
This will make the image invisible. Search for other usages of `<ScreenReaderOnly>` to see how it should be used. You got rid of the anchor tag - you'll want to bring that back.